docs/environment-variables.md

```diff
@@ -34,6 +34,7 @@ those.
   calls to make when scanning logs for a subgraph. Defaults to 100.
 - `GRAPH_ETHEREUM_MAX_EVENT_ONLY_RANGE`: Maximum range size for `eth.getLogs`
   requests that dont filter on contract address, only event signature.
+- `ETHEREUM_JSON_RPC_TIMEOUT`: Timeout for Ethereum JSON-RPC requests.
```

Comment:
I recall you being a proponent of prefixing our environment variables with `GRAPH_`. I've since stuck to the prefix for everything, let's keep using it. 😉

Reply:
We really should! Done.